argusdashboard/src/AppBundle/Controller/Configuration/GenerateController.php

```diff
@@ -0,0 +1,349 @@
+<?php
+
+namespace AppBundle\Controller\Configuration;
+
+use AppBundle\Controller\BaseController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+use AppBundle\Form\GenerateWeeklyReportType;
+use AppBundle\Form\GenerateMonthlyReportType;
+use AppBundle\Form\GenerateAlertType;
+use AppBundle\Utils\Epidemiologic;
+
+class GenerateController extends BaseController

```

Comment:
Could you add log in this class to log every anormal behaviour and errors ? Thanks